### ISSUE TITLE
Problems 1) when testing with an unused sequence of RobotCar Dataset (e.g 2014-11-14-16-34-33, a night scene) 2) and the pkl file 3) download Robotcar full sequence

### ISSUE THREAD

**Issue by jialuwang123321** [NVlabs/geomapnet#37]
Dear Sir, 
May I consult you two questions about the code?

	Q1:
I noticed from https://github.com/NVlabs/geomapnet/blob/master/README.md#other-tools that you have pre-computed mean and std for RobotCar. 

If I want to test the model **using a new data sequence of RobotCar (e.g 2014-11-14-16-34-33, a night scene),** do I need to re-run scripts/dataset_mean.py and calc_pose_stats.py? Or it is ok to use the pre-computed one in your code?

	Q2:
According to your home page, scripts/align_vo_poses.py. must be run to generate those xx_vo_stats.pkl files before using VO in MapNet++ training. 
If I **only want to train and test the mapnet module**, will those xx_vo_stats.pkl files be used? Do I still need to use run align_vo_poses.py to generate them? 

	Q3:
  According to your [comment ](https://github.com/NVlabs/geomapnet/issues/26#issuecomment-537523192), 'For each day (e.g. DAY=2014-05-14-13-46-12) you should download DAY_stereo_center.tar and DAY_vo.tar from the Bumblebee XB3 section, and DAY_gps.tar from the GPS/INS section.' 
  However, I found **multiple** stereo_centre.tar files from the full sequence (for 2014-11-28-12-07-13, **there are 6 si,ilar files stereo_centre_01.tar ~ stereo_centre_06.tar** ). **Which one should I download for your code?** Or any one of it would be fine? 

Thank you in advance!

Best wishes！
Jialu

**Comment by samarth-robo** [comment]
Hi @jialuwang123321 , these are great questions!

1. If you want to test a pre-trained model on a new sequence, you need to compute and save the pose normalization information (`mean_t` and `std_t`) by running `calc_pose_stats.py` for that sequence. However, you should not run `dataset_mean.py`. That script calculates the pixel normalization information, which should be calculated only from the training images. And we already provide that in the repository.

2. Since MapNet only uses the ground truth poses, and does not use any VO, you do not need `align_vo_poses.py` for MapNet alone.

3. You need to download and unzip **all** of them.

**Comment by jialuwang123321** [comment]
Dear Sir,
   Thank you so much for your patient help! I will have a try according to your answer!
Best
Jialu
